### ISSUE TITLE
Find Elements broken

### ISSUE THREAD

**Issue by titusfortner** [jgraham/wires#29]
The latest wires release is giving me the following in both nightly & developer

```
-> POST session/685f8c96-ec23-4f58-b76e-9d80cab6d43f/elements
   >>> http://localhost:4444/session/685f8c96-ec23-4f58-b76e-9d80cab6d43f/elements | {"using":"css selector","value":".nameC"}
<- {"error":"unknown error","message":"Expected object params"}
```

**Comment by titusfortner** [comment]
I dug through all the failures I'm seeing from the Ruby specs. I'm getting the same error for getAlertText & getWindowHandles, and findElementBy when an implicit wait is set.

**Comment by AutomatedTester** [comment]
fixed in b94ce10883c2f862f925aa9041e1c8a5b84ea1a6
